src/app/(dashboard)/categories/_components/create-category-form.tsx

```
@@ -9,22 +9,20 @@ import {
-const CreateCategorySchema = z.object({
-  name: z.string().min(1),
-  icon: z.string().min(1).emoji(),
-  color: z.string().min(1),
-});
+import { Controller, useForm } from "react-hook-form";
+import { createCategory } from "../actions";
 
-type FormData = z.infer<typeof CreateCategorySchema>;
+type CreateCategoryFormProps = {
+  // setOpen: (open: boolean) => void;
+};
```

Biome

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)